build.gradle

```diff
@@ -27,7 +28,7 @@ dependencies {
 
 intellij {
     version '2019.3.2'
-    plugins = ['java']
+    plugins = ['java', 'org.intellij.scala:2019.3.26']
```

Comment:
adding a particular version of the scala plugin (it seems to be updated once the IDE starts with `runIde`), additionally you need to complete this step: https://github.com/Aalto-LeTech/intellij-plugin/issues/65#issue-560340331

Reply:
How does this work, if a user has a different version of scala plugin already installed?

Reply:
**hm... now it does not upgrade...**
![Screenshot from 2020-02-21 11-20-55](https://user-images.githubusercontent.com/9991098/75020900-3e97f980-549c-11ea-9630-62aadbaca493.png)

Reply:
![Screenshot from 2020-02-21 11-27-54](https://user-images.githubusercontent.com/9991098/75021538-4015f180-549d-11ea-9f62-969ca8e19ee8.png)
**suggest an update, if the version is below the latest available for this build of IDEA**

Reply:
Do you know does it suggest downgrading Scala plugin if a user has a newer one? Or is it assumed that new versions are backward compatible?

Reply:
I'll check